ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -222,18 +239,29 @@ valid for [Slice], [Scope], [Service], [Socket], [Mount], or [Swap] sections
 type ${project.artifactId}_unitfile_config_systemd_resource_control = {
     'CPUAccounting' ? boolean
     'CPUShares' ? long(2..262144)
+    'CPUWeight' ? long(1..10000)
```

Comment:
It's probably worth adding a `systemd_weight` type for all the weights that are `long(1..10000)`.